uiowa/uiowa
# Admissions - Stat Fragment Tweaks and stuff

Small tweaks. Resolves https://github.com/uiowa/uiowa/issues/3012

## To test

- When adding a stat fragment on Admissions (http://admissions.local.drupal.uiowa.edu/fragment/add/stat), you will need to fill out a title, which on save, forms the title and is what is referenced on the AoS content type. The previous presave string together prefix, figure, suffix has been removed.
- There is no longer a hover effect on the additional stat details content.
- There is lovely help text for the Admissions Requirements stuff related to https://github.com/uiowa/uiowa/pull/3017
